release/build/main.go

```diff
@@ -140,49 +148,104 @@ func hashreleaseSubCommands(cfg *config.Config, runner *registry.DockerRunner) [
 			Name:  "build",
 			Usage: "Build a hashrelease locally in _output/",
 			Flags: []cli.Flag{
-				&cli.BoolFlag{Name: skipValidationFlag, Usage: "Skip pre-build validation", Value: false},
+				&cli.BoolFlag{Name: skipValidationFlag, Usage: "Skip all pre-build validation", Value: false},
+				&cli.BoolFlag{Name: skipBranchCheckFlag, Usage: "Skip check that this is a valid release branch.", Value: false},
 				&cli.BoolFlag{Name: buildImagesFlag, Usage: "Build images from local codebase. If false, will use images from CI instead.", Value: false},
 				&cli.StringFlag{Name: imageRegistryFlag, Usage: "Specify image registry to use, for development", Value: ""},
+				&cli.StringFlag{Name: operatorImageFlag, Usage: "Specify the operator image to use, for development", Value: ""},
```

Comment:
Should these have default values? How does the user know the format?

Reply:
No but I have examples of what the values can be